src/js/player.js

```diff
@@ -57,4 +62,12 @@ Player.prototype.useItem = function(pickup) {
   }
 }
 
+Player.prototype.buildHunger = function (game) {
+  this.fullness--;
+  if (this.fullness <= 0) {
+    this.health = 0;
```

Comment:
Maybe this is just to start, but I feel the actor should take damage over time instead of instantly dying.